src/sinks/clickhouse/arrow_schema.rs

```diff
@@ -0,0 +1,767 @@
+//! Schema fetching and Arrow type mapping for ClickHouse tables.
+
+use std::sync::Arc;
+
+use ::arrow::datatypes::{DataType, Field, Schema, TimeUnit};
```

Comment:
```suggestion
use arrow::datatypes::{DataType, Field, Schema, TimeUnit};
```

Don't think the `::` prefix is needed